### ISSUE TITLE
[Accessibility] DetailsList column headers not accessible when {selectionMode={SelectionMode.single}} using Keyboard navigation.

### ISSUE THREAD

**Issue by saurabhsuniljain** [OfficeDev/office-ui-fabric-react#4270]
#### Describe the issue:

When selectionMode is set to SelectionMode.single, the column headers are not accessible through Keyboard. The column header is accessible when selection mode set to either none or multiple.

#### Actual behavior:

Column header not accessible using Keyboard when selection mode is single.

#### Expected behavior:

Column header should be accessible using Keyboard when selection mode is single.

**Comment by betrue-final-final** [comment]
This will be a picker issue. @bigbadcapers

**Comment by oengusmacinog** [comment]
@saurabhsuniljain I am not experiencing the issue you describe on the latest version of fabric in Chrome on Windows.  Can you tell me the OS and browser you are experiencing this on along with the version number of Fabric React?

**Comment by oengusmacinog** [comment]
@saurabhsuniljain I am going to go ahead and close this since I am not seeing this issue, but if you are experiencing this issue still please open another issue and detail your OS and Browser so we can reproduce and debug.
